### ISSUE TITLE
new --api  results in not working docker build

### ISSUE THREAD

**Issue by yannick** [gobuffalo/buffalo#1839]
steps to reproduce:
`go get -u -v -tags sqlite github.com/gobuffalo/buffalo/buffalo` 
`buffalo new myproject --api `
`cd myproject`
`docker build .`
error:
->
`package myproject/actions: unrecognized import path "myproject/actions" (import path does not begin with hostname)`

**Comment by yannick** [comment]
the reason is that the `WORKDIR` in the Dockerfile is set to the name of the directory where `buffalo new` was run in vs. the name of the project.

**Comment by dschulz** [comment]
I just noticed the same problem. It's due to a typo in this line

```
ENV GO111MODULES=on
```

It should be `GO111MODULE` (without the 'S'):   

```
ENV GO111MODULE=on 
```

I tried to find the template or file from which the `Dockerfile` file is generated but I couldn't find any instances of `GO111MODULES` (with an 's') in it in the repo.

**Comment by dschulz** [comment]
I also replaced the line:

```
RUN go get ./...
```

for 
```
RUN go mod tidy
```

The docker image is built without problems after that, but I can't confirm this last change had any effect. The real problem is the typo.

**Comment by withakay** [comment]
The related issue #1854 has been closed but the problem is still present in v15.3.
It looks like the fix was merge to `development` but has not got further than that - https://github.com/gobuffalo/buffalo/pull/1786
